.github/workflows/run-tests-with-coverage.yml

```
@@ -36,25 +36,22 @@ jobs:
           - 5432:5432
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set up PHP
-        uses: shivammathur/setup-php@v2
+      - uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php-version }}
-          extensions: mbstring, pdo, pdo_pgsql, gd, dom, fileinfo, pgsql
-          coverage: xdebug
+          coverage: xdebug, pcov
```

let's remove the `pcov` coverage mode for all other packages as well

```suggestion
          coverage: xdebug
```